om-2.0.rdf

```diff
@@ -10276,6 +10277,18 @@
     </rdfs:subClassOf>     
   </owl:Class>
 
+  <!-- Particle Count Per Cubic Centimetre Ontology -->
+
+  <om:UnitDivision rdf:about="&om;particleCountPerCubicCentimetre">
+    <rdfs:label xml:lang="en">particle count per cubic centimetre</rdfs:label>
+    <rdfs:label xml:lang="nl">deeltjesaantal per kubieke centimeter</rdfs:label>
+    <rdfs:comment xml:lang="en">Particle count per cubic centimetre is a unit defined as the concentration of airborne particles, such as dust, pollen, pollutants, or aerosols, present in a cubic centimetre of air.</rdfs:comment>

```

Comment:
I would remove "pollutants" from this list, because there are many non-particulate pollutants...

Reply:
okay, that make sense. I have removed it.